downstream/modules/playbooks/proc-run-ansible-network-playbook.adoc

```diff
@@ -0,0 +1,157 @@
+
+When a module runs in a playbook, the output is held in memory for use by future tasks instead of written to the console. With most other modules you must explicitly register a variable to store and reuse the output of a module or task.
+
+For more information about facts, see [Ansible facts] in the _Ansiible Playbook Reference Guide_.
+
+The following debug task lets you see the results in your shell.
+
+.Procedure
+. Run the playbook with the following command.
++
+----
+ansible-playbook -i vyos.example.net, -u ansible -k -e ansible_network_os=vyos.vyos.vyos first_playbook.yml
+----
++
+The playbook contains one play with two tasks, and generates output like this.
++
+----
+$ ansible-playbook -i vyos.example.net, -u ansible -k -e ansible_network_os=vyos.vyos.vyos first_playbook.yml
+
+PLAY [Network Getting Started First Playbook]
+***************************************************************************************************************************
+
+TASK [Get config for VyOS devices]
+***************************************************************************************************************************
+ok: [vyos.example.net]
+
+TASK [Display the config]
+***************************************************************************************************************************
+ok: [vyos.example.net] => {
+    "msg": "The hostname is vyos and the OS is VyOS 1.1.8"
+}
+----
+
+. Now that you can retrieve the device configuration, you can try updating it with Ansible. 
+. Download `first_playbook_ext.yml` from link:https://docs.ansible.com/ansible/latest/_downloads/47cc11a5d29fe635cb56cb6e1cd74e0f/first_playbook_ext.yml[here], which is an extended version of the first playbook:
++
+The playbook looks like this:
++
+----
+---
+
+- name: Network Getting Started First Playbook Extended
+  connection: ansible.netcommon.network_cli
+  gather_facts: false
+  hosts: all
+  tasks:
+
+    - name: Get config for VyOS devices
+      vyos.vyos.vyos_facts:
+        gather_subset: all
+
+    - name: Display the config
+      debug:
+        msg: "The hostname is {{ ansible_net_hostname }} and the OS is {{ ansible_net_version }}"
+
+    - name: Update the hostname
+      vyos.vyos.vyos_config:
+        backup: yes
+        lines:
+          - set system host-name vyos-changed
+
+    - name: Get changed config for VyOS devices
+      vyos.vyos.vyos_facts:
+        gather_subset: all
+
+    - name: Display the changed config
+      debug:
+        msg: "The new hostname is {{ ansible_net_hostname }} and the OS is {{ ansible_net_version }}"
+----
+
+. The extended first playbook has five tasks in a single play. 
+. Run the playbook with the following command.
++
+----
+$ ansible-playbook -i vyos.example.net, -u ansible -k -e ansible_network_os=vyos.vyos.vyos first_playbook_ext.yml
+----
+
+. The output shows you the change Ansible made to the configuration:
```

Comment:
Change the sentence to not be a step (or else nets the codeblock that follows it with a `+`